src/dcc.c

```diff
@@ -1787,7 +1787,8 @@ static void dcc_telnet_pass(int idx, int atr)
       struct threaddata *td = threaddata();
       /* mark socket to read next incoming at reduced len */
       td->socklist[i].flags |= SOCK_SENTTLS;
-      dprintf(idx, "starttls\n");
+      /* Prefix with \n in case of newline-less ending stealth_prompt */
+      dprintf(idx, "\nstarttls\n");
```

Comment:
maybe only add \n in front of starttls if stealth_prompt really doesn't end with '\n' ? something like if (*stealth_prompt && stealth_prompt[len(stealth_prompt) - 1] != '\n') ?

Reply:
Unnecessary, "starttls" and "passreq" are only sent to bots, our socket reader skips all "\n" (and "\r").